pkg/controller/testutil/test_utils.go

```diff
@@ -310,14 +310,15 @@ func (m *FakeNodeHandler) Patch(_ context.Context, name string, pt types.PatchTy
 		}
 	}
 
+	logger := klog.FromContext(ctx)
 	originalObjJS, err := json.Marshal(nodeCopy)
 	if err != nil {
-		klog.Errorf("Failed to marshal %v", nodeCopy)
+		logger.Error(nil, "Failed to marshal", "node", klog.KObj(&nodeCopy))
```

Comment:
`logger` is only used in error paths, better repeat it whenever needed instead of paying the price while each call to `Patch`
```suggestion
		klog.FromContext(ctx).Error(nil, "Failed to marshal", "node", klog.KObj(&nodeCopy))
```

Reply:
I agree.

@Octopusjust: can you make that change?

Reply:
OK. I will update later :) @pohly @Namanl2001